e2e/tests/datasets/dataset-snowman-menu-actions.spec.ts

```diff
@@ -0,0 +1,117 @@
+import DataResourceCard from 'app/component/data-resource-card';
+import ExportToNotebookModal from 'app/modal/export-to-notebook-modal';
+import NotebookPreviewPage from 'app/page/notebook-preview-page';
+import WorkspaceDataPage from 'app/page/workspace-data-page';
+import { Language, LinkText, MenuOption, ResourceCard } from 'app/text-labels';
+import { makeRandomName, makeWorkspaceName } from 'utils/str-utils';
+import { findOrCreateWorkspace, signInWithAccessToken } from 'utils/test-utils';
+import DatasetRenameModal from 'app/modal/dataset-rename-modal';
+import { waitForText } from 'utils/waits-utils';
+
+describe('Datasets card snowman menu actions', () => {
+  const ProgrammingLanguage = [{ LANGUAGE: Language.R }, { LANGUAGE: Language.Python }];
```

Comment:
Nit: Can we make the var name plural

Also, will this work as a simple array since there's only one property on each object?
`const ProgrammingLanguages = [Language.R, Language.Python];`

Reply:
Array will work too. An object is expandable. Good for additions in future.
I renamed `ProgrammingLanguage ` to `KernelLanguages`, and `each` to `kernelLanguage`.
Thank you.

Reply:
Ok, that makes sense. Thanks